hl7v23/Package.md

```diff
@@ -18,7 +18,7 @@ import ballerinax/health.hl7v23
 
 | Package                       | version |
 |-------------------------------|---------|
-| ballerinax/health.hl7v2       | 1.0.5   |
+| ballerinax/health.hl7v2       | 2.0.0   |
```

Comment:
2.0.1

Reply:
Automated post-release commit bumps the version only in bal toml. The developer is expected to change this when doing the release pr.